src/App.js

```diff
@@ -1,9 +1,97 @@
 import React from 'react';
+import Button from "./components/Button";
+import Product from './components/Product';
+import FourSection from "./components/FourSection";
 import './App.css';
+import pinkBag from "./assets/bag_1.png";
```

Comment:
Iets beter is om bestandsnamen met zogenaamde 'kebab-case' te schrijven en kleine letters, i.p.v. met een underscore: bag-1.png.

Reply:
had jij m veranderd? Ik heb m letterlijk overgenomen van het huiswerk, maar goeie tip voor mijn eindopdracht

Reply:
Nee, ik had de bestanden ook niet veranderd. Dus ik ga ook mijn eigen advies opvolgen.